xml/System.Reflection/MethodImplAttributes.xml

```diff
@@ -162,7 +162,7 @@
       </ReturnValue>
       <MemberValue>512</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Specifies that the method may contain hot code and should be aggressively optimized.</summary>
```

Comment:
Value 256 states
`Specifies that the method should be inlined wherever possible`.

For consistency should we just state fot 512:
`Specifies that the method should be optimized wherever possible`

Reply:
Thanks. I will submit them in a new PR. This has already been commited.